### PR TITLE
Feature/profile ui

### PR DESCRIPTION
**Pull request by at-lytruong**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/Define/Demo.swift [hunk 1]
```diff
@@ -8,8 +8,13 @@
 
 import Foundation
 import  UIKit
-    struct Demo {
+struct Demo {
         var image: String = ""
         var name: String = ""
         var price: String = ""
     }
```
Comment (at-hieutran) [inline MyApp/Define/Demo.swift]: Remove String

File: MyApp/Define/Demo.swift [hunk 2]
```diff
@@ -8,8 +8,13 @@
...
+
+struct Profile {
+    var icon: String = ""
+    var nameIcon: String = ""
```
Comment (at-hieutran) [inline MyApp/Define/Demo.swift]: Remove String

File: MyApp/View/Controllers/Cart/CartViewController.swift [hunk 3]
```diff
@@ -12,24 +12,25 @@ class CartViewController: UIViewController {
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Cart/CartViewController.swift]: Miss MVVM.View

File: MyApp/View/Controllers/Profile/ProfileTableViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    var viewModel: ProfileCellViewModel? {
+        didSet {
+            updateView()
+        }
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        self.accessoryType = .disclosureIndicator
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Profile/ProfileTableViewCell.swift]: remove self

File: MyApp/View/Controllers/Profile/ProfileViewController.swift [hunk 5]
```diff
@@ -9,8 +9,42 @@
...
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var profileTableView: UITableView!
+    var viewModel = ProfileViewModel()
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        configTable()
+        // Remove unused seperator
+        profileTableView.tableFooterView = UIView()
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Profile/ProfileViewController.swift]: Move to configTable()

File: MyApp/View/Controllers/Profile/ProfileViewController.swift [hunk 6]
```diff
@@ -9,8 +9,42 @@
...
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeue(ProfileTableViewCell.self)
+        cell.viewModel = viewModel.viewModelForItem(at: indexPath)
+        return cell
+    }
+}
+extension ProfileViewController {
+    fileprivate func configTable() {
+
+        self.title = "Profile"
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Profile/ProfileViewController.swift]: Move to ViewDidLoad.
Remove self

File: MyApp/ViewModel/Profile/ProfileCellViewModel.swift [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+//  Created by Hải Lý on 7/27/17.
+//  Copyright © 2017 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import Foundation
+import MVVM
+
+final class ProfileCellViewModel: MVVM.ViewModel {
+    var icon: String = ""
+    var nameIcon: String = ""
```
Comment (at-hieutran) [inline MyApp/ViewModel/Profile/ProfileCellViewModel.swift]: Remove String

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**